src/pages/membership/index.tsx

```
@@ -5,15 +5,15 @@ import { styled } from "styled-components";
-import { getToken } from "../socialLogin/utils/token";
+import { getUserToken } from "../socialLogin/utils/token";
 import Contents from "./components/Contents";
 import Title from "./components/Title";
 import Curriculum from "./curriculum";
 import Event from "./event";
 import { media } from "style/responsiveStyle";
 
 export default function Membership() {
-  const token = getToken();
+  const token = getUserToken();
   const getNameResponse = token ? useGetName() : null;
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)